fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirSchemaConstants.java

```diff
@@ -60,6 +60,10 @@
     public static final String REINDEX_TSTAMP = "REINDEX_TSTAMP";
     public static final String REINDEX_TXID = "REINDEX_TXID";
     public static final String REINDEX_SEQ = "REINDEX_SEQ";
+    public static final String RESOURCE_CHANGE_LOG = "RESOURCE_CHANGE_LOG";
+
+    // Type of change C - Create, U - Update, S - Soft Delete. H - Hard Delete
+    public static final String CHANGE_TYPE = "CHANGE_TYPE";
```

Comment:
is this supposed to align with constants in ResourceChangeLogRecord ?  I didn't see the distinction between soft delete and hard delete in that one, so wasn't sure if this comment was accurate or not.

Reply:
It was a comment for the future when we support hard-delete.